hip_data_tools/connect/cassandra.py

```diff
@@ -24,10 +24,10 @@ def get_ssl_context(cert_path: str) -> SSLContext:
 @dataclass
 class CassandraConnectionSettings:
```

Comment:
why is this in this module? Shouldn't it live the cassandra stuff?

Reply:
good point.

Reply:
restructured